src/main/java/it/grid/storm/config/IniReader.java

```diff
@@ -28,78 +28,76 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.base.Preconditions;
+
 public class IniReader {
 
 	private static Logger log = LoggerFactory.getLogger(IniReader.class);
-	private final String confPath;
-	public static final String DEFAULT_CONF_PATH = Configuration.getInstance()
-		.configurationDir();
-	private List<File> iniFiles = new ArrayList<File>();
+	private final String configurationPath;
 
-	public IniReader() throws IllegalArgumentException {
+	public IniReader() {

```

Comment:
I would drop this constructor and just use the one with the parameter so that you don't need an hardcoded dependency on Configuration